src/iso_alloc.c

```diff
@@ -138,28 +138,18 @@ INTERNAL_HIDDEN void _verify_zone(iso_alloc_zone_t *zone) {
 
 INTERNAL_HIDDEN iso_alloc_root *iso_alloc_new_root(void) {
     void *p = NULL;
```

Comment:
Shouldn't `p` be a `iso_alloc_root *` instead of a `void *`?

Reply:
Doesn't matter much. Either we cast a `iso_alloc_root` as `void` or vice-versa.